README.md

```diff
@@ -30,7 +30,7 @@ the Extraction tool binary (i.e., ExtractionTool_deploy.jar) and its run script.
 
 **Step 3:** Create a folder for output files and run the Extraction tool:
 ```build
-./run-td-extract.sh -j <terajdbc4.jar> --db-address <database address> --output <output path> --db-user <db user> --db-password <db password>
+./run-td-extract.sh -j <terajdbc4.jar> --db-address <database address> --output <output path> --db-user <db user>
```

Comment:
Shouldn't this be `--db-password` w/o an argument to trigger the prompt?

Reply:
If there is a password in the argument list, run-td-extract.sh will execute the binary with that password (as before), else run-td-extract.sh will add --db-password w/o an argument to trigger the prompt. I think it's more intuitive for the customers. Do you agree?

Reply:
I see. Makes sense.